src/main.rs

```diff
@@ -27,19 +27,15 @@ struct Cat{
     status: String,
 }
 
-
 fn main() -> Result<(), Box<std::error::Error>> {
-    let mut resp = reqwest::get("https://ra-api.adoptapet.com/v1/shelters/75215/pets/?start=1&limit=100")?.text()?;
+    let mut resp =
+        reqwest::get("https://ra-api.adoptapet.com/v1/shelters/75215/pets/?start=1&limit=100")?
+            .text()?;
     let cats: Value = serde_json::from_str(&resp)?;
     println!("{:#?}", cats);
     let cats: CatList = serde_json::from_str(&resp)?;
```

Comment:
reassignment?

Reply:
Ee reassignment is ok, as long as you're using let. See https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#shadowing
If my b doesn't like as a general practice that's fair, I can switch names. lmk what you think after reading above. reassignment might still be sketchy

Reply:
"when one can one must" - a silly man

Reply:
Ok maybe not this time